docs/database-engine/install-windows/install-sql-server-from-the-installation-wizard-setup.md

```diff
@@ -479,7 +479,7 @@ This isn't applicable to [!INCLUDE[sql-server-2019](../../includes/sssql19-md.md
 1. On the **Azure Extension for SQL Server** page, you can configure SQL Server to connect to Azure. [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)] introduces this extension to enable using Azure services such as Microsoft Defender for Cloud, Microsoft Purview, Azure Active Directory and others.
 
    > [!NOTE]
-   > If you're installing SQL Server 2022 on an Azure VM, you won't see the **Azure Extension for SQL Server** page during installation. Connectivity to Azure Services for SQL Server on Azure VMs is handled through the [SQL IaaS Agent Extension](/azure/azure-sql/virtual-machines/windows/sql-server-iaas-agent-extension-automate-management), which is automatically pushed to your Azure VM shortly after SQL Server installation. You can register your VM with the extension [manually](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm) if you don't want to wait for automatic registration. 
+   > If you're installing SQL Server 2022 on an Azure VM, you won't see the **Azure Extension for SQL Server** page during installation. Connectivity to Azure Services for SQL Server on Azure VMs is handled through the [SQL IaaS Agent Extension](/azure/azure-sql/virtual-machines/windows/sql-server-iaas-agent-extension-automate-management), which is automatically pushed to your Azure VM shortly after SQL Server installation. You can register your VM with the extension [manually](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm) if you don't want to wait for automatic registration. Refer to [Supported SQL Server Versions and Operating Systems](https://learn.microsoft.com/en-us/sql/sql-server/azure-arc/prerequisites?view=sql-server-ver16&tabs=azure) for more information on supported configurations.
```

Comment:
Please use the internal linking method here:

```suggestion
   > If you're installing SQL Server 2022 on an Azure VM, you won't see the **Azure Extension for SQL Server** page during installation. Connectivity to Azure Services for SQL Server on Azure VMs is handled through the [SQL IaaS Agent Extension](/azure/azure-sql/virtual-machines/windows/sql-server-iaas-agent-extension-automate-management), which is automatically pushed to your Azure VM shortly after SQL Server installation. You can register your VM with the extension [manually](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm) if you don't want to wait for automatic registration. Refer to [Supported SQL Server Versions and Operating Systems](../../sql-server/azure-arc/prerequisites.md#supported-sql-server-versions-and-operating-systems) for more information on supported configurations.
```

Reply:
Ack